client/packages/system/src/Patient/Insurance/columns.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ColumnAlign,
+  ColumnDescription,
+  ColumnFormat,
+  useColumns,
+} from '@openmsupply-client/common';
+import { InsuranceFragment } from '../api/operations.generated';
+
+type Status = 'Active' | 'Inactive';
+
+export const useInsuranceColumns = () => {
+  const columns: ColumnDescription<InsuranceFragment>[] = [
+    {
+      label: 'label.policy-number',
+      key: 'policyNumber',
+    },
+    {
+      label: 'label.provider-name',
+      key: 'providerName',
+      accessor: ({ rowData }) => rowData.insuranceProviders?.providerName,
+    },
+    {
+      label: 'label.policy-type',
+      key: 'policyType',
+    },
+    {
+      label: 'label.discount-rate',
+      key: 'discountRate',
+      accessor: ({ rowData }) => rowData.discountPercentage,
+    },
+    {
+      label: 'label.expiry-date',
```

Comment:
I see you have sort for this in the backend, but it's not sortable in the list view, feels like would be nice to make it sortable...

Reply:
Totally missed that! Will implement